meta-balena-reference/layers/meta-balena-bsp/recipes-core/images/resin-image-initramfs.bbappend

```diff
@@ -0,0 +1 @@
+IMAGE_ROOTFS_MAXSIZE = "16384"
```

Comment:
This should definitely go into meta-balena, no need for each device type to tweak this, let;s just set one that works for all.

Reply:
This was tracked https://github.com/balena-os/meta-balena/issues/1790